packages/web/src/pages/sign-up-page/pages/SignUpPage.tsx

```diff
@@ -90,101 +92,108 @@ export const SignUpPage = () => {
-                  to={SIGN_IN_PAGE}
-                  variant='body'
-                  size='medium'
-                  strength='strong'
-                  color='secondary'
+                <Flex
+                  direction='column'
+                  gap='l'
+                  alignItems='flex-start'
+                  w='100%'
                 >
-                  {messages.signIn}
-                </Link>
-              </Text>
-            </Flex>
-          </Flex>
-        </Form>
-      )}
-    </Formik>
+                  <Text color='heading' size='l' variant='heading' tag='h1'>
+                    {messages.title}
+                  </Text>
+                  <Text color='default' size='l' variant='body' tag='h2'>
+                    {messages.subHeader}
+                  </Text>
+                </Flex>
+                <Flex
+                  direction='column'
+                  gap='l'
+                  w='100%'
+                  alignItems='flex-start'
+                >
+                  <HarmonyTextField
+                    name='email'
+                    autoFocus
+                    autoComplete='email'
+                    label={messages.emailLabel}
+                  />
+                  <Flex w='100%' alignItems='center' gap='s'>
+                    <Divider className={styles.flex1} />
+                    <Text variant='body' size='m' tag='p' color='subdued'>
+                      {messages.socialsDividerText}
+                    </Text>
+                    <Divider className={styles.flex1} />
+                  </Flex>
+                  <Flex direction='row' gap='s' w='100%'>
+                    <SocialButton
+                      socialType='twitter'
+                      className={styles.flex1}
+                      aria-label='Sign up with Twitter'
+                    />
+                    <SocialButton
+                      socialType='instagram'
+                      className={styles.flex1}
+                      aria-label='Sign up with Instagram'
+                    />
+                    <SocialButton
+                      socialType='tiktok'
+                      className={styles.flex1}
+                      aria-label='Sign up with TikTok'
+                    />
+                  </Flex>
+                </Flex>
+                <Flex
+                  direction='column'
+                  gap='l'
+                  alignItems='flex-start'
+                  w='100%'
+                >
+                  <Button
+                    variant={ButtonType.PRIMARY}
+                    type='submit'
+                    fullWidth
+                    iconRight={IconArrowRight}
+                    isLoading={isSubmitting}
+                  >
+                    {messages.signUp}
+                  </Button>
+
+                  <Text size='l'>
+                    {messages.haveAccount}{' '}
+                    {/* TODO [C-3278]: Update with Harmony Link */}
```

Comment:
I think we have this now

Reply:
FWIW the link is not dev ready, just a quick attempt to get something for harmony storybook